permissions/plugin-upload-pgyer-plugin.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "upload-pgyer"
```

Comment:
File name is inconsistent with artifact ID. Please see this repository's README what file name to use based on what the artifact ID is.

Reply:
updated